src/assets/javascripts/templates/version/index.tsx

```diff
@@ -68,7 +68,7 @@ export function renderVersionSelector(versions: Version[]): HTMLElement {
           <li class="md-version__item">
             <a
               class="md-version__link"
-              href={`${new URL(version.version, config.base)}`}
+              href={`${new URL(version.version + "/", config.base)}`}
```

Comment:
Thanks for providing the fix! Could you please use template literal syntax? We use that throughout the entire project.

```
new URL(`${version.version}/`, config.base)}
```

Reply:
Sure, looks much better!

Reply:
The build artifacts were conflicting with master, so decided not to include them. Please let me know if they should be.